src/pages/iou/MoneyRequestTagPage.js

```diff
@@ -109,6 +109,14 @@ export default compose(
             key: ONYXKEYS.IOU,
         },
     }),
+    //  eslint-disable-next-line
```

Comment:
Please update all of these to use the format for only disabling the specific rule `no-multiple-onyx-in-file`

Reply:
Noted, It's already updated to // eslint-disable-next-line rulesdir/no-multiple-onyx-in-file